site/package.json

```diff
@@ -44,7 +44,9 @@
     "svgo-loader": "2.2.2",
     "unist-util-visit": "^2.0.3",
     "wcag-contrast-verifier": "^1.1.0",
-    "webpack": "^4.46.0"
+    "webpack": "^4.46.0",
+    "react-media": "^1.10.0",
+    "@kaizen/brand": "^1.1.5"
```

Comment:
Mistake here? These are already included above.

Reply:
Yeah, github auto update added them. Let me fix it manually.